doc/api/net.md

```diff
@@ -529,6 +674,9 @@ The optional `callback` parameter will be executed when the data is finally
 written out - this may not be immediately.
 
 ## net.connect(options[, connectListener])
+<!-- YAML
+added: v0.1.90
+-->
```

Comment:
Hmm… 0.5.10? I don’t think this was exported directly from `net` before (ditto for the ones below).